index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Angular Application</title>
+</head>
+<body>
+
+    <div ng-app="app">
+        <div ng-controller="MainController">
+            Hello, {{ name }}!
+        </div>
+    </div>
+
+    <script src="js/angular.js"></script>
+    <script src="js/app/app.js"></script>
+    <script src="js/app/controllers/MainController.js"></script>
```

Comment:
Sure - I'll post it to a branch called 'ian-error-solution'. The error says
that 'MainController' is not defined when I pass it into the controller
function in 'app.js'

On Tue, Jan 12, 2016 at 8:44 PM, Todd Motto <notifications@github.com>
wrote:

> In index.html
> <https://github.com/learn-co-curriculum/angular-project-setup-lab/pull/1#discussion_r49542358>
> :
>
> > +    <meta charset="utf-8">
> > +
> > +    <meta http-equiv="X-UA-Compatible" content="IE=edge">
> > +    <title>Angular Application</title>
> > +</head>
> > +<body>
> > +
> > +    <div ng-app="app">
> > +        <div ng-controller="MainController">
> > +            Hello, {{ name }}!
> > +        </div>
> > +    </div>
> > +
> > +    <script src="js/angular.js"></script>
> > +    <script src="js/app/app.js"></script>
> > +    <script src="js/app/controllers/MainController.js"></script>
>
> Would you mind posting the setup? It should work as is, the module system
> in Angular is designed this way, only module setters need initialising
> before all module extensions. Quick jsfiddle or something would be great
> (or email me your code!)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learn-co-curriculum/angular-project-setup-lab/pull/1/files#r49542358>
> .
>



-- 


     Ian Candy
     Web Instructor